### PR TITLE
apictl integration tests

### PR DESCRIPTION
**Pull request by uvindra**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: import-export-cli/integration/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# APICTL Integration testing
+
+## Pre-requisites for running integration tests
+1. In order to run the integration tests, two instances of API Manager need to be started. These two instances will act as the APIM environments that will be used for doing exporting and importing scenarios respectively. The integration tests will execute apictl commands, DCR calls and REST API calls against these instances.
+
+2. Before starting the APIM instances update the APIM *deployment.toml* to allow the `@` character as part of the service provider name. This is to support DCR for tenant users since the `@` is part of the username. The updated ocnfig should look as follows,
```
Comment (npamudika) [inline import-export-cli/integration/README.md:6]: spelling mistake in config

File: import-export-cli/integration/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# APICTL Integration testing
+
+## Pre-requisites for running integration tests
+1. In order to run the integration tests, two instances of API Manager need to be started. These two instances will act as the APIM environments that will be used for doing exporting and importing scenarios respectively. The integration tests will execute apictl commands, DCR calls and REST API calls against these instances.
```
Comment (npamudika) [inline import-export-cli/integration/README.md:4]: Shall we mention the two instances are running with port offset 0 and 1 here as we have mentioned that is the default in the step 3.

File: import-export-cli/integration/version_test.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/base"
+)
+
+func TestVersion(t *testing.T) {
+	output, _ := base.Execute(t, "version")
+
+	assert.True(t, strings.Contains(output, "3.1.0"), "Expected version is 3.1.0, actual version is "+output)
```
Comment (npamudika) [inline import-export-cli/integration/version_test.go:32]: Thinking whether it is good to hard-code the version here as we will be using the integration tests future with the other APIM versions too. WDYT?

File: import-export-cli/integration/README.md (hunk 1 above)
Comment (uvindra) [reply import-export-cli/integration/README.md]: Fixed in https://github.com/wso2/product-apim-tooling/pull/310/commits/18287b957368281440c9d4da82ebaf51636407e2

File: import-export-cli/integration/README.md (hunk 2 above)
Comment (uvindra) [reply import-export-cli/integration/README.md]: Fixed in https://github.com/wso2/product-apim-tooling/pull/310/commits/18287b957368281440c9d4da82ebaf51636407e2

File: import-export-cli/integration/version_test.go (hunk 3 above)
Comment (uvindra) [reply import-export-cli/integration/version_test.go]: +1, fixed in https://github.com/wso2/product-apim-tooling/pull/310/commits/4d5f9c8e487f96ba96bd053d08f428d91199de0b

**Pull request merged**